src/app/shopping-list/page.tsx

```diff
@@ -1,4 +1,48 @@
+import { ScrollArea } from "@/components/ui/scroll-area";
+
+import ShoppingBagIcon from "@/public/icons/shopping-bag.svg";
+
+import { CartItem } from "@/components/ShoppingCart/CartItem";
+import { type CartItem as CartItemType } from "@/components/ShoppingCart/types";
+import Image from "next/image";
+
+const itemsList: CartItemType[] = [
+  {
+    name: "Eggs",
+    quantity: 12,
+    goTo: "/products/eggs",
+    done: false,
```

Comment:
I know that we will get this from DB, but for what is this "done"? As a check, that I get an item in the shop?